src/gui.rs

```diff
@@ -28,41 +29,20 @@ mod simulation_settings;
 mod tabs;
 mod theme;
 mod top_bar;
+pub mod windows; // TODO: make this private (it is public because it has ARCHIVE)
 
 use crate::data_source::*;
-use crate::file::*;
 use crate::settings::AppSettings;
 use crate::simulation::*;
 use crate::state::*;
 
+use crate::gui::components::top_menu_bar::create_top_menu_bar;
 use crate::gui::fc_settings::*;
 use crate::gui::simulation_settings::*;
 use crate::gui::tabs::*;
 use crate::gui::theme::*;
 use crate::gui::top_bar::*;
-
-// Log files included with the application. These should probably be fetched
-// if necessary to reduce application size.
-// TODO: migrate old launches
-pub const ARCHIVE: [(&str, Option<&'static str>, Option<&'static str>); 5] = [
-    ("Zülpich #1", None, None),
-    ("Zülpich #2", None, None),
-    (
-        "DARE (FC A)",
-        Some("https://raw.githubusercontent.com/tudsat-rocket/sam/main/archive/dare_launch_a_telem_filtered.json"),
-        Some("https://raw.githubusercontent.com/tudsat-rocket/sam/main/archive/dare_launch_a_flash_filtered.json"),
-    ),
-    (
-        "DARE (FC B)",
-        Some("https://raw.githubusercontent.com/tudsat-rocket/sam/main/archive/dare_launch_b_telem_filtered.json"),
-        Some("https://raw.githubusercontent.com/tudsat-rocket/sam/main/archive/dare_launch_b_flash_filtered.json"),
-    ),
-    (
-        "EuRoC 2023 (ÆSIR Signý)",
-        Some("https://raw.githubusercontent.com/tudsat-rocket/sam/main/archive/euroc_2023_telem_filtered.json"),
-        Some("https://raw.githubusercontent.com/tudsat-rocket/sam/main/archive/euroc_2023_flash_filtered.json"),
-    ),
-];
+use crate::gui::windows::archive::open_archive_window;
```

Comment:
Definitely a fan of moving this somewhere else. In the future, I'd like to have some sort of Archive struct that is a bit smarter about things like caching, but that would probably be independent of the Window. Also like the structure with a separate windows folder. We might have more of these in the future.

Reply:
I totally agree, it's more predictable and more similar to how big UI are built